python/pyspark/sql/connect/client.py

```diff
@@ -632,7 +632,7 @@ def _execute(self, req: pb2.ExecutePlanRequest) -> None:
             Proto representation of the plan.
 
         """
-        logger.info("Execute")
+        logger.debug(f"Execute request: » {os.linesep}{req}{os.linesep} «")
```

Comment:
I would avoid the characters such as `»` that's not used in the current codebase.

Reply:
curious: why?

Reply:
Can you suggest a better alternative?

Reply:
Maybe just `Execute request: {os.linesep}{req}{os.linesep}`? I would stick to the existing codebase unless it needs a reason.

It's also sort of documented:

> If you’re not sure about the right style for something, try to follow the style of the existing codebase

https://spark.apache.org/contributing.html

So I wouldn't do something new unless there's a reason.